### PR TITLE
Router, models, tracker endpoints

### PR DESCRIPTION
**Pull request by pachkovska**
This PR includes the following changes I am proposing:
1. Added router. Proposing to use route prefixes - it is a standard practice.
2. Separated Models out of the main file as well as endpoints
3. Modified and added to Mashas endpoints

### CODE REVIEW

File: models/trackers.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel
+from typing import Optional
+from datetime import datetime
+
+
+class Tracker(BaseModel):
+    name: Optional[str]
+    url_address: str
+    created_at: Optional[datetime] = datetime.now()
+    is_offer: Optional[bool] = None
```
Comment (mloyanich) [inline models/trackers.py:10]: I would remove this from the model, or put frequency instead. Even though we won't be using it now, we know that we will definitely need it in the future. We could set it optional and put 24 as the default value.

File: api/routes/trackers.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+def read_tracker_by_id(tracker_id: int):
+    if tracker_id in trackers.keys():
+        return trackers[tracker_id]
+    return {"message": "Tracker not found"}
+
+
+@router.post("/")
+def create_tracker(tracker: Tracker):
+    tracker_id = random.randint(1, 9999)
+    trackers[tracker_id] = tracker
```
Comment (mloyanich) [inline api/routes/trackers.py:24]: before we assign, we should check whether there is no tracker with the same randomly generated id

File: api/routes/users.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    return False
+
+
+class User(BaseModel):
+    first_name: str
+    last_name: str
+    email: str
+
+
+@router.post("/create-user/{user_id}")
```
Comment (pachkovska) [inline api/routes/users.py:22]: @allar14 User_id needs to be generated automatically. You generally do not pass id as a parameter when you are creating new entity record.

Comment (mloyanich) [reply api/routes/users.py:22]: @allar14 , you can take a look at how the POST method is done for /trackers router

Comment (allar14) [reply api/routes/users.py]: updated the code addressing your comments above

**Review by mloyanich — APPROVED** [review]
Looks good!

**Pull request merged**